run/preview.go

```diff
@@ -29,9 +30,9 @@ Table: {{.Name}}({{$schema}}.{{.DBName}})
 {{end -}}
 `))
 
-// Preview displays the database info that woudl be passed to your template
+// Preview displays the database info that would be passed to your template
 // based on your configuration.
-func Preview(env environ.Values, cfg *Config, useYaml bool) error {
+func Preview(env environ.Values, cfg *Config, format string) error {
```

Comment:
I'd prefer if this used an enum so it was clear what acceptable values are.

Reply:
Hey, an enum might seem nice but i have a feeling it will be an overkill. A honest switch with three branches quite fits well here. 

We are only expecting `yaml`, `tabular` or `types` no need to be fancy!